Simplified/Audiobooks/LCP/LCPAudiobooks.swift

```diff
@@ -0,0 +1,116 @@
+//
+//  LCPAudiobooks.swift
+//  Simplified
+//
+//  Created by Vladimir Fedorov on 16.11.2020.
+//  Copyright © 2020 NYPL Labs. All rights reserved.
+//
+
+#if LCP
+
+import Foundation
+import R2Shared
+import R2Streamer
+import NYPLAudiobookToolkit
+
+/// LCP Audiobooks helper class
+@objc class LCPAudiobooks: NSObject {
+    
+  private let audiobookUrl: URL
+  private let lcpService = LCPLibraryService()
+
+  /// .lcpa archive container
+  private let container: Container
+
+  /// Distributor key - one can be found in `NYPLBook.distributor` property
+  @objc static let distributorKey = "lcp"
+  
+  /// Initialize for an LCP audiobook
+  /// - Parameter audiobookUrl: must be a file with `.lcpa` extension
+  @objc init?(for audiobookUrl: URL) {
+    self.audiobookUrl = audiobookUrl
+    do {
+      guard let (pubBox, _) = try Publication.parse(at: self.audiobookUrl) else {
+        return nil
+      }
+      self.container = pubBox.associatedContainer
+    } catch {
+      NYPLErrorLogger.logError(error, summary: "Error opening audiobook")
+      return nil
+    }
+  }
+  
+  /// Content dictionary for `AudiobookFactory`
+  @objc func contentDictionary() -> NSDictionary? {
+    let manifestPath = "manifest.json"
+    do {
+      // Relative path inside the audiobook
+      let data = try container.data(relativePath: manifestPath)
+      let publicationObject = try JSONSerialization.jsonObject(with: data, options: .allowFragments) as? NSDictionary
+      return publicationObject
+    } catch {
+      NYPLErrorLogger.logError(error, summary: "Error reading audiobook \(manifestPath)")
```

Comment:
Instead of `manifestPath` (which is a constant and doesn't really add value) can you add something that specifies this is an LCP error? This also in the `init` at line 38

Reply:
sure, added "LCP" in both cases

Reply:
added logging with "LCP" text to `decrypt` function as well